doc/source/user_guide/indexing.rst

```diff
@@ -1158,6 +1158,39 @@ Mask
    s.mask(s >= 0)
    df.mask(df >= 0)
 
+.. _indexing.np_where:
+
+Setting with enlargement conditionally using :func:`numpy`
+----------------------------------------------------------
+
+An alternative to :meth:`~pandas.DataFrame.where` is to use :func:`numpy.where`.
+Combined with setting a new column, you can use it to enlarge a dataframe where the
+values are determined conditionally.
+
+When you have two choices to choose from, say in the following dataframe, set the color
+to 'green' when the second column has 'Z'.  You can do the following:
```

Comment:
Probably this phrasing will be slightly simpler to comprehend?

```
Consider you have two choices to choose from in the following dataframe.
To set the color to 'green' when the second column contains 'Z', you can do the following.
```

But maybe not, just a comment.

Reply:
I reworded it, but slightly differently.  Let me know what you think :)

Reply:
Sounds good. Thank you!